<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MacBook-Air-3097.local" failures="3" tests="24" name="Command line test" time="88.365" errors="0" timestamp="2022-01-16T19:01:18 EET">
  <testcase classname="tests.TestLogin" name="testCreateLoginOtp" time="7.205"/>
  <testcase classname="tests.TestLogin" name="testVerifyLoginOtp" time="2.769"/>
  <testcase classname="tests.transactionsTests.TestUserTransactionsIntegration" name="testCreateTransaction" time="1.575"/>
  <testcase classname="tests.transactionsTests.TestUserTransactionsIntegration" name="testGetUserTransactions" time="1.405"/>
  <testcase classname="tests.transactionsTests.TestUserTransactionsIntegration" name="testGetNftTransactions" time="1.815"/>
  <testcase classname="tests.transactionsTests.TestUserTransactionsIntegration" name="testUpdateTransaction" time="1.473"/>
  <testcase classname="tests.transactionsTests.TestUserTransactionsIntegration" name="testDeleteTransaction" time="1.707"/>
  <testcase classname="tests.transactionsTests.TestUserTransactionsIntegration" name="verifyTheTransactionStatusAfterDeleting" time="1.544"/>
  <testcase classname="tests.contactsTests.TestContacts" name="testCreateContact" time="1.574">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;200&amp;gt; but was &amp;lt;500&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <500>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at helpers.ContactsServiceHelper.createContact(ContactsServiceHelper.java:32)
at tests.contactsTests.TestContacts.testCreateContact(TestContacts.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 52 stack frames]]>
    </failure>
  </testcase> <!-- testCreateContact -->
  <testcase classname="tests.contactsTests.TestContacts" name="testGetContactsOfAnUser" time="0.0">
    <skipped/>
  </testcase> <!-- testGetContactsOfAnUser -->
  <testcase classname="tests.contactsTests.TestContacts" name="testImportOneOrMoreContacts" time="0.0">
    <skipped/>
  </testcase> <!-- testImportOneOrMoreContacts -->
  <testcase classname="tests.contactsTests.TestContacts" name="testGetContact" time="0.0">
    <skipped/>
  </testcase> <!-- testGetContact -->
  <testcase classname="tests.contactsTests.TestContacts" name="testUpdateContact" time="0.0">
    <skipped/>
  </testcase> <!-- testUpdateContact -->
  <testcase classname="tests.contactsTests.TestContacts" name="testDeleteContact" time="0.0">
    <skipped/>
  </testcase> <!-- testDeleteContact -->
  <testcase classname="tests.contactsTests.TestContacts" name="verifyTheContactStatusAfterDeleting" time="0.0">
    <skipped/>
  </testcase> <!-- verifyTheContactStatusAfterDeleting -->
  <testcase classname="tests.usersTests.TestUserIntegration" name="testGetUnauthorizedUser" time="0.765"/>
  <testcase classname="tests.usersTests.TestUserIntegration" name="testGetUser" time="0.804"/>
  <testcase classname="tests.usersTests.TestUserIntegration" name="testGetUserWithProvidingEmptyUserId" time="0.969"/>
  <testcase classname="tests.usersTests.TestUserIntegration" name="testGetWrongUser" time="0.841"/>
  <testcase classname="tests.usersTests.TestUserIntegration" name="testUpdateUser" time="1.609"/>
  <testcase classname="tests.usersTests.TestUserIntegration" name="testUserAttributes" time="0.783">
    <failure type="java.lang.AssertionError" message="The Full Name is empty expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The Full Name is empty expected [false] but found [true]
at tests.usersTests.TestUserIntegration.testUserAttributes(TestUserIntegration.java:45)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 40 stack frames]]>
    </failure>
  </testcase> <!-- testUserAttributes -->
  <testcase classname="tests.nftsTests.TestNfts" name="testGetAllNfts" time="1.973">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected status code &amp;lt;200&amp;gt; but was &amp;lt;500&amp;gt;.
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected status code <200> but was <500>.

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:126)
at io.restassured.specification.ResponseSpecification$statusCode$0.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:134)
at io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:89)
at io.restassured.internal.ValidatableResponseImpl.super$2$statusCode(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy:142)
at io.restassured.internal.ValidatableResponseImpl.statusCode(ValidatableResponseImpl.groovy)
at helpers.NftServiceHelper.getAllNftDetails(NftServiceHelper.java:43)
at tests.nftsTests.TestNfts.testGetAllNfts(TestNfts.java:41)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 52 stack frames]]>
    </failure>
  </testcase> <!-- testGetAllNfts -->
  <testcase classname="tests.nftsTests.TestNfts" name="testGetSingleNft" time="0.816"/>
  <testcase classname="tests.nftsTests.TestNfts" name="testUpdateNft" time="2.456"/>
</testsuite> <!-- Command line test -->
